app/models/bookings/placement_date_subject.rb

```diff
@@ -9,6 +9,13 @@ class Bookings::PlacementDateSubject < ApplicationRecord
     inclusion: { in: :allowed_subject_choices },
     if: :bookings_subject_id
 
+  # this combined id is used when candidates are choosing a subject specific
+  # date and are presented with a single list of options that contains subjects
+  # AND placement dates
+  def combined_id
+    [bookings_subject.id, id].join("_")
```

Comment:
should this be bookings_placement_date.id?

Reply:
🤦🏽‍♂️ Thanks

Fixed and added a spec covering it